
Attaching package: 'cowplot'

The following object is masked from 'package:ggplot2':

    ggsave

Loading required package: foreach
Loading required package: iterators
Loading required package: parallel

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

starting worker pid=102664 on localhost:11589 at 10:08:55.858
starting worker pid=102675 on localhost:11589 at 10:08:56.094
starting worker pid=102686 on localhost:11589 at 10:08:56.327
starting worker pid=102697 on localhost:11589 at 10:08:56.564
starting worker pid=102708 on localhost:11589 at 10:08:56.803
starting worker pid=102719 on localhost:11589 at 10:08:57.043
starting worker pid=102730 on localhost:11589 at 10:08:57.281
starting worker pid=102741 on localhost:11589 at 10:08:57.517
starting worker pid=102752 on localhost:11589 at 10:08:57.752
starting worker pid=102763 on localhost:11589 at 10:08:57.988
starting worker pid=102774 on localhost:11589 at 10:08:58.222
starting worker pid=102785 on localhost:11589 at 10:08:58.454
starting worker pid=102796 on localhost:11589 at 10:08:58.688
starting worker pid=102807 on localhost:11589 at 10:08:58.927
starting worker pid=102818 on localhost:11589 at 10:08:59.163
starting worker pid=102829 on localhost:11589 at 10:08:59.398
starting worker pid=102840 on localhost:11589 at 10:08:59.633
starting worker pid=102851 on localhost:11589 at 10:08:59.872
starting worker pid=102862 on localhost:11589 at 10:09:00.104
starting worker pid=102873 on localhost:11589 at 10:09:00.338
starting worker pid=102884 on localhost:11589 at 10:09:00.572
starting worker pid=102895 on localhost:11589 at 10:09:00.808
starting worker pid=102906 on localhost:11589 at 10:09:01.042
starting worker pid=102917 on localhost:11589 at 10:09:01.275
starting worker pid=102928 on localhost:11589 at 10:09:01.510
starting worker pid=102939 on localhost:11589 at 10:09:01.745
starting worker pid=102950 on localhost:11589 at 10:09:01.980
starting worker pid=102961 on localhost:11589 at 10:09:02.214
starting worker pid=102972 on localhost:11589 at 10:09:02.446
starting worker pid=102983 on localhost:11589 at 10:09:02.678
[1] "warning, sym=2, snp=3: 0 sample"
[1] "warning, sym=1, snp=1: 0 sample"
[1] "warning, sym=3, snp=2: 0 sample"
[1] "warning, sym=2, snp=2: 0 sample"
[1] "warning, sym=3, snp=6: 0 sample"
[1] "warning, sym=3, snp=3: 0 sample"
[1] "warning, sym=4, snp=4: 0 sample"
[1] "warning, sym=5, snp=5: 0 sample"
[1] "warning, sym=5, snp=7: 0 sample"
[1] "warning, sym=6, snp=3: 0 sample"
[1] "warning, sym=6, snp=6: 0 sample"
[1] "warning, sym=7, snp=7: 0 sample"
[1] "warning, sym=7, snp=5: 0 sample"
[1] "warning, sym=8, snp=8: 0 sample"
[1] "warning, sym=9, snp=9: 0 sample"
[1] "warning, sym=10, snp=10: 0 sample"
[1] "warning, sym=11, snp=11: 0 sample"
[1] "warning, sym=12, snp=12: 0 sample"
[1] "warning, sym=13, snp=13: 0 sample"
[1] "warning, sym=14, snp=14: 0 sample"
Error in { : task 1 failed - "could not find function "erf""
Calls: source ... eval -> eval -> condfreqstatistics -> %dox% -> <Anonymous>
In addition: Warning message:
In e$fun(obj, substitute(ex), parent.frame(), e$data) :
  already exporting variable(s): savedir, filename, data, symptoms, namesymptoms, symptomvariants, numsymptomvariants, snps, namesnps, snpvariants, numsnpvariants, logpriortheta, statsfunction, mciterations, mcdiscard
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
srun: error: compute-0-3: task 0: Exited with exit code 1
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
